operator/pkg/securedcluster/scanner/auto_sense_test.go

```diff
@@ -18,16 +19,26 @@ var securedCluster = platform.SecuredCluster{
 	},
 }
 
+var validClusterVersion = &unstructured.Unstructured{
```

Comment:
nit: The validClusterVersion is duplicated, you could move this to `operator/pkg/utils/testutils` as an exposed variable to reduce duplicate code in `const.go`.